runtime/browser/demo/explorer.html

```diff
@@ -0,0 +1,28 @@
+<link rel="import" href="../../../strategy-explorer/strategy-explorer.html"></link>
```

Comment:
I think this file should be under
arcs/runtime/browser/lib/
or maybe we want to create
arcs/runtime/browser/lib/demo 
with common files used for different demos (and move demo-base.js) there.
because this folder contains for the gift-shopping demo files specifically.

also i think explorer.html is a bit too generic.
but i don't have a good suggestion, because there already is strategy-explorer.html and strategy-explorer-demo.html (the latter demos the capabilities of the strategy explorer with a static demo recipe)